CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Sourcery CHANGELOG
 
 ---
-## Master
+## 0.16.2
```

Comment:
Usually we update this during the release process 🤔
I would keep the `Master` here and release the `0.16.2` after merging this.